internal/pkg/deploy/cloudformation/project_test.go

```diff
@@ -1022,7 +1022,7 @@ func TestDelegateDNSPermissions(t *testing.T) {
 				return &mockCloudFormation{
 					t: t,
 					mockCreateChangeSet: func(t *testing.T, in *cloudformation.CreateChangeSetInput) (*cloudformation.CreateChangeSetOutput, error) {
-						require.Equal(t, len(in.Parameters), 4)
+						require.Equal(t, len(in.Parameters), 5)
```

Comment:
nit: can we swap the order here as well just like in the other test :P

Reply:
👍